docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,174 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
```

Comment:
@catong 👍 I like that idea.

@srpearce That is news to me, but the naming conventions for these various APIs makes talking about them a dubious task, so I'm not surprised that there is some confusion.  Let's get together on monday and nail this down.